sark/dpkg.py

```diff
@@ -19,9 +19,11 @@
 _source_ts = ["csv", "xls", "xlsx"]  # "sqlite"
 _pd_types = {
```

Comment:
What motivates this list? I.e. why is 'yearmonth' but not 'yearweek', 'yearday' included?

Reply:
It does raise the general issue with hardcoding, I suppose, since it's difficult to imagine the possible permutations of timeseries

Reply:
This comes from the [frictionless data format](https://specs.frictionlessdata.io/table-schema/#types-and-formats), not up to us to decide :-|.

Reply:
I should make a note about this in the context of custom datetime format.